### PR TITLE
[FIX] error on applying multi-theme with snippets on few websites

### PR DESCRIPTION
**Pull request by yelizariev**
closes #410

### CODE REVIEW

File: website_multi_theme/models/ir_ui_view.py [hunk 1]
```diff
@@ -45,3 +45,22 @@ def _customize_template_get_views(self, key, full=False, bundles=False):
...
+            website_id = self.env.context['search_multi_website_snippet']
+            domain += [
+                '|',
+                ('website_id', '=', website_id),
+                ('website_id', '=', False)
+            ]
+            _logger.debug('Updated domain: %s', domain)
+
+        res = super(IrUiView, self).search(
+            domain, offset=offset, limit=limit, order=order, count=count)
```
Comment (yajo) [inline website_multi_theme/models/ir_ui_view.py]: I think you're forgetting to patch `order`

**Review by yajo — CHANGES_REQUESTED** [review]

File: website_multi_theme/models/ir_ui_view.py (hunk 1 above)
Comment (pedrobaeza) [reply website_multi_theme/models/ir_ui_view.py]: What do you mean?

Comment (yajo) [reply website_multi_theme/models/ir_ui_view.py]: I suggested [this](https://github.com/yelizariev/odoo/commit/52de9ae545506c8601e727c0cc80b52b4cce3707#r28729620) to him, and he put it [here](https://github.com/odoo/odoo/pull/24429/files#diff-0662b757930550c519324fd45a444369R54)

Comment (pedrobaeza) [reply website_multi_theme/models/ir_ui_view.py]: Well, it's an enhancement there for giving more priority to specific website. OK then.

Comment (yelizariev) [reply website_multi_theme/models/ir_ui_view.py]: Fixed and covered https://github.com/yelizariev/website/commit/4816f516a289a9ee57eb3b7186d065add1e29d8a

File: website_multi_theme/models/ir_ui_view.py [hunk 2]
```diff
@@ -45,3 +45,23 @@ def _customize_template_get_views(self, key, full=False, bundles=False):
...
+                ('website_id', '=', False)
+            ]
+            order = 'website_id DESC'
+            _logger.debug('Updated domain: %s', domain)
+
+        res = super(IrUiView, self).search(
+            domain, offset=offset, limit=limit, order=order, count=count)
+
+        if self.env.context.get('search_multi_website_snippet'):
+            return res[:1]
```
Comment (yajo) [inline website_multi_theme/models/ir_ui_view.py]: you can also patch `limit = 1` above.

**Review by yajo — APPROVED** [review]

**Comment by yelizariev** [comment]
Improved by using limit:
https://github.com/yelizariev/website/commit/ea30bd12771c1e0a42b57482d984a19eb6fa7c9f

Travis is read due to timeout issue. I made a separate PR https://github.com/OCA/website/pull/455

**Comment by pedrobaeza** [comment]
Please rebase to get green Travis

**Review by rafaelbn — APPROVED** [review]
Thanks! :+1: 

Please still pending travis :red_circle:

**Comment by rafaelbn** [comment]
@yelizariev I like you see here! more OCA man! :smile:

**Comment by yelizariev** [comment]
Rebased and travis is green :green_heart:

**Pull request merged**